hardware_interface/test/test_handle.cpp

```diff
@@ -15,7 +15,10 @@
 #include <thread>
 
 #include "gmock/gmock.h"
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #include "hardware_interface/handle.hpp"
+#pragma GCC diagnostic pop
```

Comment:
May be this is not needed. Let me check this afternoon and confirm

Reply:
we should be cleaning these up in rolling/kilted anyways soon, let's move